cartridges/int_externalCustomerData/cartridge/scripts/utils/urlUtils.js

```diff
@@ -0,0 +1,17 @@
+
+
+var host = "https://json-server-app-707ded616226.herokuapp.com/";
```

Comment:
This should go into the service definition (credentials.)

Reply:
fixed